src/main/webapp/help-GerritAbortCurrentBuilds.html

```diff
@@ -0,0 +1,4 @@
+<p>Sets <b>abort current builds</b> mode.</p>
+
+<p>If this is enabled, all currently running builds for previous patchsets
```

Comment:
It should be a sub-setting to the cancel setting, it is quite hard to grok what it does as is. But from what I can see it is a subset of that setting. And the help text should explain that as well.